src/main/java/com/google/api/generator/gapic/composer/ServiceClientSampleCodeComposer.java

```diff
@@ -289,6 +290,114 @@ public static String composeRpcDefaultMethodHeaderSampleCode(
+    VariableExpr clientVarExpr =
+        VariableExpr.withVariable(
+            Variable.builder()
+                .setName(JavaStyle.toLowerCamelCase(clientType.reference().name()))
+                .setType(clientType)
+                .build());
+    // Assign method's request variable with the default value.
+    VariableExpr requestVarExpr =
+        VariableExpr.withVariable(
+            Variable.builder().setName("request").setType(method.inputType()).build());
+    Message requestMessage = messageTypes.get(method.inputType().reference().simpleName());
+    Preconditions.checkNotNull(requestMessage);
+    Expr requestBuilderExpr =
+        DefaultValueComposer.createSimpleMessageBuilderExpr(
+            requestMessage, resourceNames, messageTypes);
+    AssignmentExpr requestAssignmentExpr =
+        AssignmentExpr.builder()
+            .setVariableExpr(requestVarExpr.toBuilder().setIsDecl(true).build())
+            .setValueExpr(requestBuilderExpr)
+            .build();
+
+    List<Expr> bodyExprs = new ArrayList<>();
+    bodyExprs.add(requestAssignmentExpr);
+
+    // Create OperationFuture variable expr with invoking client's lro callable method.
+    // e.g. OperationFuture<Empty, WaitMetadata> future =
+    //          echoClient.waitOperationCallable().futureCall(request);
+    TypeNode operationFutureType =
+        TypeNode.withReference(
+            ConcreteReference.builder()
+                .setClazz(OperationFuture.class)
+                .setGenerics(
+                    method.lro().responseType().reference(),
+                    method.lro().metadataType().reference())
+                .build());
+    VariableExpr operationFutureVarExpr =
+        VariableExpr.withVariable(
+            Variable.builder().setName("future").setType(operationFutureType).build());
+    MethodInvocationExpr rpcMethodInvocationExpr =
+        MethodInvocationExpr.builder()
+            .setExprReferenceExpr(clientVarExpr)
+            .setMethodName(
+                String.format("%sOperationCallable", JavaStyle.toLowerCamelCase(method.name())))
+            .build();
+    rpcMethodInvocationExpr =
+        MethodInvocationExpr.builder()
+            .setExprReferenceExpr(rpcMethodInvocationExpr)
+            .setMethodName("futureCall")
+            .setArguments(requestVarExpr)
+            .setReturnType(operationFutureType)
+            .build();
+    bodyExprs.add(
+        AssignmentExpr.builder()
+            .setVariableExpr(operationFutureVarExpr.toBuilder().setIsDecl(true).build())
+            .setValueExpr(rpcMethodInvocationExpr)
+            .build());
+
+    List<Statement> bodyStatements =
+        bodyExprs.stream().map(e -> ExprStatement.withExpr(e)).collect(Collectors.toList());
+    bodyExprs.clear();
+    // Add a line comment
+    bodyStatements.add(CommentStatement.withComment(LineComment.withComment("Do something;")));
+
+    // Assign response variable with invoking client's lro method.
```

Comment:
Nit: LRO in all caps, same throughout in comments.

Reply:
SG